packages/expo-secure-store/android/src/main/java/expo/modules/securestore/SecureStoreModule.kt

```diff
@@ -133,13 +133,16 @@ open class SecureStoreModule : Module() {
     try {
       when (scheme) {
         AESEncryptor.NAME -> {
-          val secretKeyEntry = getPreferredKeyEntry(SecretKeyEntry::class.java, mAESEncryptor, options, requireAuthentication, usesKeystoreSuffix)
-            ?: throw DecryptException("Could not find a keychain for key $key$legacyReadFailedWarning", key, options.keychainService)
+          val secretKeyEntry = getKeyEntryCompat(SecretKeyEntry::class.java, mAESEncryptor, options, requireAuthentication, usesKeystoreSuffix) ?: run {
+            Log.w(TAG, "An entry was found for key $key under keychain ${options.keychainService}, but there is no corresponding KeyStore key. The value will be removed to avoid future errors. Returning null")
```

Comment:
```suggestion
            Log.w(TAG, "An entry was found for key $key under keychain ${options.keychainService}, but there is no corresponding KeyStore key. This situation occurs when the app is reinstalled. The value will be removed to avoid future errors. Returning null")
```

nit: maybe mention how could the case happening in the log.